language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/TransactionClientImpl.java

```diff
@@ -75,40 +75,6 @@ public Flowable<Transaction> getTransactions(
     return getTransactions(begin, end, filter, verbose, Optional.of(accessToken));
   }
 
-  private <Ct> Flowable<Ct> getContracts(
```

Comment:
I have delete this utility added in #14969.  It doesn't make too much sense as it filter out archive events and convert the created event to a contract type